frontend/test/pytest/test_measurements_results.py

```diff
@@ -625,22 +649,63 @@ def circuit(x, y, coeffs):
         )
 
 
-class TestOtherMeasurements:
-    """Test other measurement processes."""
+class TestState:
+    """Test state measurement processes."""
+
+    def test_state_on_0qbits(self, backend):
+        """Test state on 0 qubits."""
+
+        @qjit
+        @qml.qnode(qml.device(backend, wires=0))
+        def state_0qbit():
+            return qml.state()
+
+        expected = np.array([complex(1.0, 0.0)])
+        observed = state_0qbit()
+        assert np.array_equal(observed, expected)
 
-    def test_state(self, backend):
-        """Test state."""
+    def test_state_on_1qubit(self, backend):
+        """Test state on 1 qubit."""
 
         @qjit
         @qml.qnode(qml.device(backend, wires=1))
         def state(x: float):
             qml.RX(x, wires=0)
             return qml.state()
 
-        expected = np.array([complex(1.0, 0.0), complex(0.0, 0.0)])
-        observed = state(0.0)
+        expected = np.array([complex(1.0, 0.0), complex(0.0, -1.0)]) / np.sqrt(2)
+        observed = state(np.pi / 2)
+        assert np.allclose(observed, expected)
+
+
+class TestProbs:
+    """Test probabilities measurement processes."""
+
+    def test_probs_on_0qbits(self, backend):
+        """Test probs on 0 qubits."""
+
+        @qjit
+        @qml.qnode(qml.device(backend, wires=0))
+        def probs_0qbit():
+            return qml.probs()
+
+        expected = np.array([1.0])
+        observed = probs_0qbit()
```

Comment:
This follows from how probabilities are calculated. Since they are obtained from the statevector, and a zero dimensional statevector has a single element of size 1, the probability for that element is also 1.